### ISSUE TITLE
Low results on validation set

### ISSUE THREAD

**Issue by ly19965** [ucas-vg/TinyBenchmark#28]
Hi, I test my model on all train images (without corner) and got 11.1 map on tiny 50. However, the same model I submitted to the challenge website on the test set can achieve 50 map on tiny 50. So, would you share you your config which test on the test  or train set without corner info (no need for merge file)

**Comment by yinglang** [comment]
just remove MERGE_RESULTS  and MERGE_GT_FILE will be ok, the necessary setting is:
TEST:
  COCO_EVALUATE_STANDARD: 'tiny'  # tiny need change
  # MERGE_RESULTS: true
  # MERGE_GT_FILE: '/home/hui/dataset/tiny_set/annotations/task/tiny_set_test_all.json'
  IGNORE_UNCERTAIN: true
  USE_IOD_FOR_IGNORE: true
